testground/benchmark/benchmark/sendtx.py

```diff
@@ -4,26 +4,36 @@
 import web3
 from eth_account import Account
 
-from .utils import export_eth_account, send_transaction
+from .utils import (
+    broadcast_tx_json,
+    build_batch_tx,
+    export_eth_account,
+    send_transaction,
+)
 
 TEST_AMOUNT = 1000000000000000000
 GAS_PRICE = 1000000000
 
 
-def fund_test_accounts(w3, from_account, num_accounts) -> [Account]:
+def fund_test_accounts(cli, w3, from_account, num_accounts, **kwargs) -> [Account]:
     accounts = []
+    sender = from_account.address
+    nonce = w3.eth.get_transaction_count(sender)
+    txs = []
     for i in range(num_accounts):
         acct = Account.create()
         tx = {
             "to": acct.address,
             "value": TEST_AMOUNT,
             "gas": 21000,
             "gasPrice": GAS_PRICE,
+            "nonce": nonce + i,
         }
-        receipt = send_transaction(w3, tx, from_account, wait=True)
-        assert receipt.status == 1
-        print("fund test account", acct.address, "balance", TEST_AMOUNT)
+        txs.append(tx)
         accounts.append(acct)
+    cosmos_tx, _ = build_batch_tx(w3, cli, txs, from_account, **kwargs)
```

Comment:
Gocha, but fund step is just preparing new accounts to send txs, doesn't matter if batch or not right? since I tried large concurrent send like `gen /tmp/data 4 1 testplan-{index} 2700 1` to avoid invalid nonce case.